### ISSUE TITLE
Create "add reward" command for testing critter drops

### ISSUE THREAD

**Issue by cyanpill** [Segs/Segs#797]
## Feature Summary
A new slash command /addreward that will call a function to give the rewards a player will get from defeating a critter, in preparation for combat.

### Is your feature request related to a problem? 
A continuation of the existing commands /addboost /addinsp /setinf /setxp, this will test giving all those rewards to a player simultaneously.

### Proposed Implementation
- Grant xp and inf based on a level [(xp chart)](https://paragonwiki.com/wiki/Experience) [(more xp)](https://paragonwiki.com/wiki/Talk:Experience/Archive_1)
- Enhancements and Inspirations will be a random selection, with level argument determining the enhancement level, and if it should be TO, DO or SO [(chance of types)](https://paragonwiki.com/wiki/Enhancements) [(insp chart)](https://paragonwiki.com/wiki/Talk:Inspirations)
- More research to find more data
- How will the data be stored?

### Alternatives and Additional Remarks

- An origin argument to limit enhancements to be that type of SO, or one of the two DO
- A percentage argument for lowering rewards for multiple players 
- Parties will give take one share, add a multiplier, then gives shares based on level proportions

**Comment by ZopharShinta** [comment]
Was there anything that limited the type of DO/SO enhancement you would get?
Like CoT were more likely to give Magic enhancements or just random from all enemy groups?

**Comment by cyanpill** [comment]
Yes, groups were more likely to give one origin. I found this thread debating about it, I'm not sure a consensus was ever reached.
http://web.archive.org/web/20090217231416/http://boards.cityofheroes.com/showflat.php?Cat=&Number=976413&page=0&view=collapsed&sb=5&o=&fpart=1#976413
